src/app/features/registration/registration.component.html

```diff
@@ -1 +1,76 @@
-<p>registration works!</p>
+<div>
```

Comment:
General question: why all inputs have a red border by default?

Reply:
fixed, styles added. Everything is working correctly now